dcli/lib/src/process/process/process_in_isolate.dart

```diff
@@ -27,18 +31,19 @@ void startIsolate(ProcessSettings settings, ProcessChannel channel) {
 SendPort _connectSendPort(ProcessChannel channel) {
   /// take the initial message which contains
   /// the channels sendPort id.
-  final msg = channel.send.takeOneMessage();
+  final msg = channel.send.take();
+
   return NativeCalls.connectToPort(msg);
 }
 
 /// Starts an isolate that spawns the command.
 void _startIsolate(ProcessSettings processSettings, ProcessChannel channel) {
-  unawaited(Isolate.spawn((mailboxAddrs) async {
+  unawaited(Isolate.spawn<List<Sendable<Mailbox>>>((mailboxes) async {
```

Comment:
Strong type to accept list of Sendable i.e. send and response.

Reference here: https://github.com/dart-lang/native_synchronization/blob/fc4ee3f3b5cdd439ed4c18f2003468ceb24e6458/lib/mailbox.dart#L130